service/internal/db/db.go

```diff
@@ -106,30 +108,27 @@ func (d SQLClient) DeleteProjectEntry(ctx context.Context, project string) error
 	return sess.WithContext(ctx).Collection(ProjectEntryDB).Find("project", project).Delete()
 }
 
-func (d SQLClient) CreateTokenEntry(ctx context.Context, project string, secretAccessor string) (TokenEntry, error) {
-	res := TokenEntry{}
-
+func (d SQLClient) CreateTokenEntry(ctx context.Context, token types.Token) error {
 	sess, err := d.createSession()
 	if err != nil {
-		return res, err
+		return err
 	}
 	defer sess.Close()
 
 	err = sess.WithContext(ctx).Tx(func(sess db.Session) error {
-		res = TokenEntry{
-			CreatedAt: time.Now().UTC().Format(time.RFC3339),
-			ProjectID: project,
-			TokenID:   secretAccessor,
+		res := TokenEntry{
+			CreatedAt: token.CreatedAt,
```

Comment:
Before we were generating the TS and it's time wouldn't be far off from Vault's creation time, but instead opted to use the creation time coming from Vault when generating a token.